webpack/webpack.config.server.js

```diff
@@ -3,7 +3,13 @@ import webpack from 'webpack';
 import HappyPack from 'happypack';
 
 const root = process.cwd();
-const serverInclude = [path.join(root, 'src', 'server'), path.join(root, 'src', 'universal')];
+const serverInclude = [
+  path.join(root, 'src', 'server'),
+  path.join(root, 'src', 'universal')
+];
+const serverExclude = [
```

Comment:
upon closer review, if there are no plans to tie migrations into the runtime (which there shouldn't be, right?), then we don't even need to exclude it, since webpack just crawls the dependencies starting with the entry point. if nothing up the tree needs it, it's safe to say it'll never be bundled.